webapp/channels/src/plugins/export.js

```diff
@@ -42,6 +42,7 @@ window.ReactRouterDom = require('react-router-dom');
 window.PropTypes = require('prop-types');
 window.Luxon = require('luxon');
 window.StyledComponents = require('styled-components');
+window.ReactSelect = require('react-select');
```

Comment:
Heads up for @hmhealey . I don't think he will have any problem with it, but just in case.

Reply:
Thanks for the heads up, that one's actually not a good one to expose because the web app's version of it is severely out of date, and it technically doesn't even support the current version of React that we use. I'm not actually sure if it'll work 100% because the package is shipped as multiple modules like `react-select` and `react-select/async`. I don't think the async one would be included this way.

This does seem like it would be a good candidate to share with plugins, but I don't think it's a good idea to start that with such an outdated version

Reply:
@hmhealey Would you like to remove the export or update the dependency? For what it's worth, it's working fine in https://github.com/mattermost/mattermost-plugin-calls/pull/820. I don't think exporting should cause problems for the webapp and no plugin other than Calls would be using this, realistically speaking. I think the plugin side should understand what version they are importing against when developing.

Reply:
It'd be quicker to remove the export for now. I looked into upgrading, but it's a few major versions behind with a number of breaking changes.

As far as I know, Calls already bundles its own version of the library, so it shouldn't cause any problems if it continues using that version